### ISSUE TITLE
Extensions: settings action missing title

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52619]
**Environment Details**:
VSCode Version : 1.24.1
OS Version : WIN10
**Additional Details:**
Keyboard,MAS1.1.1
**Repro Steps:**
1. Launch VS Code.
2. Navigate to View(Alt+V) menu option and select "Extensions" from dropdown.
3. Navigate to setting  button under treeview.
4. Check whether Tooltip provided.

**Actual:**
Tooltip is not provided to setting button.

**Expected:**
Tooltip should be provided to button.
**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx](url)

**User Impact:**
Keyboard user unable to know more  information about the button.
**MAS Reference:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}](url)

Does this issue occur when all extensions are disabled?: Yes
**Attachment For Reference**
[[Accessibility]A11y_VSCode_ViewExtension_Keyboard3.pptx](https://github.com/Microsoft/vscode/files/2124028/Accessibility.A11y_VSCode_ViewExtension_Keyboard3.pptx)

**Comment by isidorn** [comment]
This is a simple adding of a .title on a div in the extensions view.
Assigning to July, @sandy081 let me know if you need help
